test/passing/tests/docstrings_toplevel_directives.mlt

```diff
@@ -0,0 +1,11 @@
+(** Header *)
+
+#use "something"
+
+let two = 2
+
+[@@@warning "-labels-omitted"] ;;
```

Comment:
This `;;` seems unnecessary as well ?

Reply:
Apparently it is, I get a syntax error from the compiler if I remove them.

Reply:
It is! Sorry about that.